packages/internal/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
Wonder if we shouldn't publish NEWS for this since it's not meant to be consumed directly